packages/viewer/src/components/scene-tree-table-layout/scene-tree-table-layout.tsx

```diff
@@ -426,13 +391,16 @@ export class SceneTreeTableLayout {
         colIndex === 0
           ? (depth: number) => `calc(${depth} * 0.5rem)`
           : () => `0`;
-      pool.iterateElements((el, binding, rowIndex) => {
-        const row = this.stateMap.viewportRows[rowIndex];
 
-        if (isLoadedRow(row)) {
-          this.updateCell(row, el, binding, rowIndex, cellPaddingLeft);
-        }
-      });
+      if (!this.isScrolling) {
```

Comment:
I think this is fine for a first pass to address the immediate issue, but is not an ideal fix. If we can't keep pace with rendering while scrolling, there's something fundamentally wrong that I think we should investigate as part of this ticket.